### ISSUE TITLE
sysadm not started on boot and crashes when attempting to start

### ISSUE THREAD

**Issue by jungle-boogie** [trueos/trueos-core#312]
Hi All,

Evidently this might be a known issue but I'm reporting it anyhow.
See here for details from Ken:
https://gitter.im/trueos/Lobby?at=588a271bfba5bfea06913d78

sysadm does not start a boot and when attempting to run `sudo service sysadm stop ; sudo killall sysadm-binary ; sudo service sysadm start` this constantly floods dmesg with various PIDs:

```
pid 3867 (sysadm-binary), uid 0: exited on signal 10
pid 3870 (sysadm-binary), uid 0: exited on signal 10
pid 3872 (sysadm-binary), uid 0: exited on signal 10
pid 3874 (sysadm-binary), uid 0: exited on signal 10
pid 3876 (sysadm-binary), uid 0: exited on signal 10
pid 3878 (sysadm-binary), uid 0: exited on signal 10
pid 3881 (sysadm-binary), uid 0: exited on signal 10
pid 3883 (sysadm-binary), uid 0: exited on signal 10
pid 3885 (sysadm-binary), uid 0: exited on signal 10
pid 3887 (sysadm-binary), uid 0: exited on signal 10
pid 3889 (sysadm-binary), uid 0: exited on signal 10
pid 3891 (sysadm-binary), uid 0: exited on signal 10
pid 3893 (sysadm-binary), uid 0: exited on signal 10
pid 3895 (sysadm-binary), uid 0: exited on signal 10
pid 3897 (sysadm-binary), uid 0: exited on signal 10
pid 3899 (sysadm-binary), uid 0: exited on signal 10
pid 3901 (sysadm-binary), uid 0: exited on signal 10
pid 3903 (sysadm-binary), uid 0: exited on signal 10
pid 3905 (sysadm-binary), uid 0: exited on signal 10
pid 3908 (sysadm-binary), uid 0: exited on signal 10
pid 3910 (sysadm-binary), uid 0: exited on signal 10
pid 3912 (sysadm-binary), uid 0: exited on signal 10
pid 3914 (sysadm-binary), uid 0: exited on signal 10
pid 3916 (sysadm-binary), uid 0: exited on signal 10
pid 3918 (sysadm-binary), uid 0: exited on signal 10
pid 3920 (sysadm-binary), uid 0: exited on signal 10
pid 3922 (sysadm-binary), uid 0: exited on signal 10
pid 3924 (sysadm-binary), uid 0: exited on signal 10
pid 3926 (sysadm-binary), uid 0: exited on signal 10
pid 3928 (sysadm-binary), uid 0: exited on signal 10
pid 3930 (sysadm-binary), uid 0: exited on signal 10
pid 3932 (sysadm-binary), uid 0: exited on signal 10
pid 3935 (sysadm-binary), uid 0: exited on signal 10
pid 3937 (sysadm-binary), uid 0: exited on signal 10
pid 3939 (sysadm-binary), uid 0: exited on signal 10
pid 3941 (sysadm-binary), uid 0: exited on signal 10
```

on build `FreeBSD 12.0-CURRENT #18 4f888bf(drm-next): Wed Jan 18 14:31:26 UTC 2017`

**Comment by jungle-boogie** [comment]
On a fresh reboot, this appears in dmesg:
`pid 1938 (sysadm-binary), uid 0: exited on signal 10`

**Comment by pkgdemon** [comment]
Can you provide the output of the ```about``` command?  Your kernel info is not matching up with what I show on my laptop for STABLE.

**Comment by jungle-boogie** [comment]
Sure!

```
% about
[: =: unexpected operator
  ______                   ____  _____
 /_  __/______  _____     / __ \/ ___/
  / / / ___/ / / / _ \   / / / /\__ \
 / / / /  / /_/ /  __/  / /_/ /___/ /
/_/ /_/   \__,_/\___/   \____//____/

General info:
       Host:...............emberg
       User:...............sean
       Uptime:.............11:12
       FreeBSD kernel ver:.12.0-CURRENT
       FreeBSD world ver:..12.0-CURRENT
       FreeBSD git branch:.drm-next
       FreeBSD git rev:....4f888bf
       TrueOS ver:.........TrueOS-Desktop-201701182017
       Arch:...............amd64
       Kernel ident:.......GENERIC
       Bootloader:.........BSD
       Bootloader type:....BIOS
       CPU:................Intel(R) Core(TM) i3-5020U CPU @ 2.20GHz
       CPU cores:..........4
       Memory (free/avail):3456 / 4096 Mb
       Package set:........UNSTABLE
       Desktop environment:Lumina
       X11 Driver:.........modesetting_drv.so
       OSS Driver:.........pcm1: <Realtek (0x0233) (Internal Analog)> (play/rec) default
       WIFI Driver:........ath0
```

**Comment by pkgdemon** [comment]
Found an issue with the supervisor tonight.  

Imported a patch from openrc:
https://github.com/trueos/freebsd/commit/73a6ea939734c00f56e39da611b1cfb9b383f941

Added new stop routines for sysadm here:
https://github.com/trueos/sysadm/commit/8080a83e1bbe8d9cbb36953222c5d3a58593141e

Prior to this supervisor was improperly tracking the pid.  I have confirmed with the patches any start, stop issues should be fixed.
